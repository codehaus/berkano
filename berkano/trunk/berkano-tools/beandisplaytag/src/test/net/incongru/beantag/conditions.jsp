<%@ page import="net.incongru.beantag.SomeObject"%>
<%@ taglib prefix="b" uri="/target/xdoclet/META-INF/taglib.tld"%>
<%
SomeObject someObject = new SomeObject("Some Value", "Other Value", 123);
pageContext.setAttribute("test.obj", someObject);
%>
<b:beandisplay name="test.obj" label="Some Object">
 <b:property name="someProperty"/>
 <b:property name="anotherProperty" label="Another Property"/>
 <b:property name="integerProperty" test="3 > 5"/>
</b:beandisplay>

<b:beandisplay name="test.obj" label="Some Object">
 <b:property name="someProperty" test="2+2==4"/>
 <b:property name="anotherProperty" label="Another Property" test="2+2==5"/>
 <b:property name="integerProperty"/>
</b:beandisplay>

<b:beandisplay name="test.obj" label="Some Object">
 <b:property name="someProperty" test="integerProperty==123"/>
 <b:property name="anotherProperty" label="Another Property" test="anotherProperty=='ThisValue'"/>
 <b:property name="integerProperty"/>
</b:beandisplay>

<b:beandisplay name="test.obj" label="Some Object With Conditional Style">
 <b:property name="someProperty" test="integerProperty==123"/>
 <b:property name="anotherProperty" label="Another Property" test="anotherProperty=='ThisValue'"/>
 <b:property name="integerProperty" test="integerProperty==123" conditionalClass="foo"/>
</b:beandisplay>
