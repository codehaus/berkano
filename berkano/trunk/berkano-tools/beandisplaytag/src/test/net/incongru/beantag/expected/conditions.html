<table>
<caption>Some Object</caption>
<tbody>
<tr class="odd">
  <td>someProperty</td>
  <td>Some Value</td>
</tr>
<tr class="even">
  <td>Another Property</td>
  <td>Other Value</td>
</tr>
</tbody>
</table>

<table>
<caption>Some Object</caption>
<tbody>
<tr class="odd">
  <td>someProperty</td>
  <td>Some Value</td>
</tr>
<tr class="even">
  <td>integerProperty</td>
  <td>123</td>
</tr>
</tbody>
</table>

<table>
<caption>Some Object</caption>
<tbody>
<tr class="odd">
  <td>someProperty</td>
  <td>Some Value</td>
</tr>
<tr class="even">
  <td>integerProperty</td>
  <td>123</td>
</tr>
</tbody>
</table>

<table>
<caption>Some Object With Conditional Class</caption>
<tbody>
<tr class="odd">
  <td>someProperty</td>
  <td>Some Value</td>
</tr>
<tr class="even foo">
  <td>integerProperty</td>
  <td>123</td>
</tr>
</tbody>
</table>
